pkg/registry/core/pod/strategy.go

```diff
@@ -94,6 +94,8 @@ func (podStrategy) Validate(ctx context.Context, obj runtime.Object) field.Error
 	}
 	allErrs := validation.ValidatePodCreate(pod, opts)
 	allErrs = append(allErrs, validation.ValidateConditionalPod(pod, nil, field.NewPath(""))...)
+	allErrs = append(allErrs, validation.ValidateSeccompAnnotationsAndFields(pod, field.NewPath(""))...)
```

Comment:
is there a reason not to call this from ValidatePodCreate (and ValidatePodUpdate), rather than requiring callers to add additional validation calls to get the entire pod validated?